birdie/src/lib/components/files/TreeNode.svelte

```diff
@@ -56,12 +56,6 @@
 		loading = false;
 	};
 
-	onMount(async () => {
-		loading = true;
-		await getChildren();
-		loading = false;
-	});
-
```

Comment:
Turns out this on mount was overwriting child nodes and losing all their children, resulting in only the top level node being applied properly